packages/kotti-ui/source/kotti-inline-edit-2/types.ts

```diff
@@ -0,0 +1,24 @@
+import escape from 'lodash/escape'
+import { z } from 'zod'
+export const DEFAULT_POST_PARSER = (_: any): any => _
+export namespace KottiInlineEdit2 {
+	export const propsSchema = z.object({
+		dangerouslyOverrideParser: z
+			.function()
+			.args(z.string())
+			.returns(z.string())
+			.default(escape),
+		placeholder: z.string().default('Click to edit'),
```

Comment:
Should be
```suggestion
		placeholder: z.string().nullable().default(null),
```
with `useI18n` inside the component to translate this string.

Consider the translation optional, but I think the `null` should be included already and the component should at least hard-code the `Click to edit` inside its own source code instead, so that it can be easily added in a future improvement

Reply:
already implemented this locally